apps/src/studio/studio.js

```diff
@@ -2660,6 +2752,36 @@ Studio.displayScore = function() {
   score.setAttribute('visibility', 'visible');
 };
 
+Studio.displayFloatingScore = function(changeValue) {
```

Comment:
Perhaps document above this function that this level flag should only be enabled if the level creator wants the score to appear from the sprite with index 0?